src/app/analytics/analytics.factory.spec.js

```diff
@@ -659,10 +663,11 @@ describe('analytics factory', () => {
             payment_type: 'credit card',
             purchase_number: '23032',
             campaign_code: undefined,
-            designation: 'designation',
+            designation: '0643021',
             item_brand: 'STAFF',
             processingFee: undefined,
             testing_transaction: 'false',
+            job_id: utmTerm,
```

Comment:
Can you add a test for when `utmTerm` isn't present?

Reply:
Just did. Thanks